apps/civicsignalblog/src/payload/components/allowedAppSelect/index.tsx

```diff
@@ -32,7 +32,7 @@ function CustomSelectComponent({ path, label }) {
         path={path}
         name={path}
         options={options}
-        value={String(value)}
+        value={value ? String(value) : ``}
```

Comment:
Why do we need `String(value)` instead of just `value`?

Reply:
It was more of trying to overcome Typescript not knowing if value is really a string but I have added proper typing on the useField hook